src/base/static/libs/maps/index.js

```diff
@@ -1,12 +1,17 @@
 import MapboxGLProvider from "./mapboxgl-provider";
 // Import other providers here as they become available
 
-var Util = require("../../js/utils.js");
-var LayerView = require("mapseed-layer-view");
+import emitter from "../../utils/emitter";
+import { createGeoJSONFromCollection } from "../../utils/collection-utils";
+
+import constants from "../../constants";
+
+const Util = require("../../js/utils.js");
 
 class MainMap {
   constructor({ container, places, router, mapConfig }) {
     this.mapConfig = mapConfig;
```

Comment:
nit: worth keeping in mind, we should distinguish between "private" and "public" fields for our es6 classes. In js, a common pattern is to prefix all fields with `_`, and using getters to access the public fields.

ie:
```
get mapConfig () { return this._mapConfig; }
```

(assuming that `mainMap.mapConfig` will be accessed from outside of this class)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get